### PR TITLE
Update Tabs nav example to match core

### PR DESCRIPTION
**Pull request by cmwylie19**
https://github.com/patternfly/patternfly-react/issues/2607

**What**: updated the Tabs.md file to make the react example match the html example.

**Additional issues**:

Was a pleasure!

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Tabs/Tabs.md [hunk 1]
```diff
@@ -449,22 +449,63 @@ class TabsNavVariant extends React.Component {
...
-        aria-label="Local"
-        variant={TabsVariant.nav}
-      >
-        <Tab eventKey={0} title="Tab item 1">
-          Tab 1 section
+       <React.Fragment>
+        <Tabs activeKey={this.state.activeTabKey} onSelect={this.handleTabClick} leftScrollAriaLabel="scrollLeft" rightScrollAriaLabel="right">
+          <Tab eventKey={0} title="Tab item 1">
+            Tab 1 section
         </Tab>
```
Comment (dlabrecq) [inline packages/patternfly-4/react-core/src/components/Tabs/Tabs.md]: The `</Tab>` tags are not indented properly.

**Comment by jgiardino** [comment]
Thanks for working this issue, @cmwylie19! 

One of our goals with the React component examples is that they align with the core HTML examples. Part of this issue is to bring the React examples [[1]](https://www.patternfly.org/v4/documentation/react/components/tabs/#tabs-using-the-nav-element) [[2]](https://www.patternfly.org/v4/documentation/react/components/tabs/#secondary-tabs-using-the-nav-element) into closer alignment with the [HTML example](https://www.patternfly.org/v4/documentation/core/components/tabs#tabs-using-the-nav-element) which shows how to use `<a>` html elements with the component.

As part of this issue, the react examples **Secondary tabs using the nav element** [[1]](https://www.patternfly.org/v4/documentation/react/components/tabs/#tabs-using-the-nav-element)  and **Tabs using the nav element** [[2]](https://www.patternfly.org/v4/documentation/react/components/tabs/#secondary-tabs-using-the-nav-element) should demonstrate using `href="#"` on the `<Tab>` components to result in using the `<a>` html element instead of `<button>` for the tabs (i.e. `.pf-c-tabs__button`).

Additionally, I see that the **Tabs using the nav element** includes an additional set of tabs that is not using `variant={TabsVariant.nav}`. I'm not sure these extra tabs are needed to demonstrate this behavior, but I also don't have a strong preference. I'm fine with either removing them, or updating them to be consistent with the other examples that use `<nav>` as the `.pf-c-tabs` element.

File: packages/patternfly-4/react-core/src/components/Tabs/Tabs.md (hunk 1 above)
Comment (cmwylie19) [reply packages/patternfly-4/react-core/src/components/Tabs/Tabs.md]: Thanks for the feedback, will work to address your two concerns.

**Comment by cmwylie19** [comment]
Hi @jgiardino, it is a pleasure.

> As part of this issue, the react examples Secondary tabs using the nav element [1] and Tabs using the nav element [2] should demonstrate using href="#" on the <Tab> components to result in using the <a> html element instead of <button> for the tabs (i.e. .pf-c-tabs__button).

Regarding the statement above, I did gathered that was your intention from the comment on the issue but i did not address it directly because I did not know you would be reading it, so sorry about that! First, let me give you some clarity around your question.

So Tabs component does have two buttons, but just to scroll left and to scroll right depending on the VP, the child of the scroll buttons are an unordered list. The unordered list contains function components: These function components decide to render an anchor or button element _solely based on the href prop that they receive_. If they are passed the href prop, they will render an anchor, if not it will be a button.

`const Component = (props.href ? 'a' : 'button') as any;`

So to address your question on whether or not you can declarative use a an anchor element instead of the button element the answer is no, not declaratively , but you can make the Tab component render the anchor tab if you feed is the href prop.

Now, i build a component that utilized href as a prop, and added a hash to the end of the url and it worked flawlessly, I wanted to add that also to the demo, exactly as it was in the html example, but the Markdown file that we are using show the examples did not play nice with the href prop. It ruined the demo in the page by freezing component state as soon as the hash was added to the end of the URL.

So, all in all, you can totally make Tab render as an anchor if you just add the prop href.

`
<Tab eventKey={2} title="Tab item 3" href="#forexample">
          Tab 3 section
</Tab>`

**Comment by jgiardino** [comment]
Thanks for the explanation, @cmwylie19 😄 You actually articulate a really great issue, that I'm really stumped on how to move forward. 

Ideally, our examples would illustrate for consumers how to use components. When I look at the react examples for Tabs, there's no example or information on how to render the tabs as links. And I question if consumers would realize that putting `href` on a `<Tab>` would work without seeing an example or having some documentation to explain that? 

I'm open to suggestions on how to move forward on this.

**Comment by dlabrecq** [comment]
FYI, we spoke during today's PatternFly standup and the suggestion was to try changing the href in the example to point at patternfly.org instead of using a hash tag -- hope that works for everyone

**Comment by dlabrecq** [comment]
Check out the text and tab examples. 

The text example uses a hash tag:
```
<Text component={TextVariants.a} href="#">
  non ante
</Text>
```

While the tabs example uses:
```
      <Tab id="tab3" eventKey={2} title="Tab item 3" href="#/items/3">
        Tab 3 section
      </Tab>
```

There are more examples with breadcrumbs, link button, alert, etc., which also use hash tags.

**Comment by mcoker** [comment]
The examples still don’t match core - first example uses `<button>`, not `<a>`, and the second example uses`<a>`, but the tabs link to google (instead of opening tab sections), and there is no secondary tabs in the second example like there is in core.

**Pull request closed without merging**